ctod.dd

```diff
@@ -266,8 +266,7 @@ for (i = 0; i &lt; ARRAY_LENGTH; i++)
 $(H4 The D Way)
 
 ----------------------------
-int array[17];
-array[] = value;
+int[17] array = value;
```

Comment:
> array[] = 7; // or array = 7;

I'm for `array[] = value;`. The other variant only works with fixed-size arrays again.